extensions/amp-consent/validator-amp-consent.protoascii

```diff
@@ -50,9 +50,29 @@ tags: {  # <amp-consent>
   html_format: AMP
   tag_name: "AMP-CONSENT"
   requires_extension: "amp-consent"
-  attr_lists: "extended-amp-global"
+  unique: true
   requires: "amp-consent extension .json script"
+  excludes: "amp-consent [type]"
+  attr_lists: "extended-amp-global"
+  amp_layout: {
+    supported_layouts: NODISPLAY
+  }
+}
+
+tags: {  # <amp-consent> (type)
+  html_format: AMP
+  tag_name: "AMP-CONSENT"
+  spec_name: "amp-consent [type]"
+  requires_extension: "amp-consent"
   unique: true
+  requires: "meta name=amp-consent-blocking"
```

Comment:
doesnt seem to work. validator is reporting error although zhouyx <meta name=amp-consent-blocking ...> is in the page

Reply:
@janwinkler could you share the page source? 

I've tried including `<meta name="amp-consent-blocking" content="amp-ad">` to the document, and it passed validation check.

Reply:
it seems its not documented, hence i only tried <meta name="amp-consent-blocking" content="">. might be good to add this to the amp-consent documentation ;-)

Reply:
@zhouyx Do you remember why we want amp-consent elements with `type` attributes to always have a meta tag?

Reply:
https://github.com/ampproject/amphtml/issues/17742#issuecomment-512564655